### PR TITLE
Add even more info, fix grammar

### PR DESCRIPTION
**Pull request by Secret-chest**

### CODE REVIEW

File: README.in.md [hunk 1]
```diff
@@ -14,10 +14,11 @@ In the modern Minecraft era, with Fabric's community effort, modders have begun
 ## This list
 
 This list will list recommended OptiFine alternatives by me (LambdAurora), so it has some requirements for mods to be listed here:
- - They must be publicly available, if it's only accessible through Discord they are not eligible.
- - Mods must respect the Minecraft's EULA.
- - Mods must not refer to a cheat client.
+ - They must be publicly available, if they're only available on request they are not eligible.
+ - Mods must respect the Mojang EULA.
+ - Cheat ("PVP") clients (Badlion / Lunar / etc.) are not considered mods.
```
Comment (LambdAurora) [inline README.in.md:20]: Unsure.

The original line is more in line with what I meant, and I would consider this change as a new line.

**Comment by Secret-chest** [comment]
@LambdAurora Can you merge now?

**Comment by Secret-chest** [comment]
@LambdAurora ?

**Comment by LambdAurora** [comment]
Sorry for not having answered directly, I would have preferred if the rules were kept in the same order as before as moving the referring to cheat clients rule to the first rule is odd to me.

**Comment by Secret-chest** [comment]
@LambdAurora Fixed

**Pull request merged**